src/Microsoft.ML.TimeSeries/Deseasonality/MeanDeseasonality.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace Microsoft.ML.TimeSeries.Deseasonality
+{
```

Comment:
Why does this have a different namespace than the other classes derived from `DeseasonalityBase`?

Reply:
Updated.

---
In reply to: [442522683](https://github.com/dotnet/machinelearning/pull/5202#discussion_r442522683) [](ancestors = 442522683)